### DISCUSSION TITLE
What is the official Orbot f-droid repo?

### DISCUSSION THREAD

**Discussion by serrq** [guardianproject/orbot-android#821] [Q&A]
Hi, usually I install foss apps from f-droid.
Would be better if I have the official repo, not that general of f-droid.

If possible both .onion and normal one.

**Comment by salad612** [comment]
See here https://guardianproject.info/fdroid/repo/

**Reply by serrq** [reply]
Fingerprint?

**Reply by salad612** [reply]
if you go to the website and hold on the link or qr code there it should let you copy the link with the fingerprint.

**Reply by serrq** [reply]
Thanks you!
